pom.xml

```diff
@@ -356,9 +355,52 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo #517:240min pending tasks in fixing findbugs.
```

Comment:
@sebing please create 30 min puzzle. Next person either will be smart enough to fix them in 30 min or will create new puzzle if needed.